src/test/java/core/basesyntax/model/BalanceOperationTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Warehouse;
+import core.basesyntax.model.entities.Fruit;
+import core.basesyntax.model.impl.BalanceOperation;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private static final Warehouse<Fruit> WAREHOUSE = new Warehouse<>();
```

Comment:
instead of creating and passing the new Storage each time, we can clear it after each test where needed